### DISCUSSION TITLE
Playing sounds on Spike Prime hub

### DISCUSSION THREAD

**Discussion by dsplaisted** [pybricks/support#1772] [General]
Hi,

I'd like to be able to play arbitrary sound files on the Spike Prime hub.  Is this possible?  I see that #929 has some ideas for some improvements, but I'm not sure what's possible today or how to use it.

Thanks,
Daniel

**Comment by dlech** [comment]
Nothing has really changed since then. Only beeping is possible with Pybricks currently.

**Comment by EclecticInventor** [comment]
Being able to play sound files would be super useful
